app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -62,6 +62,11 @@ angular.module('Bastion.components').factory('Nutupane',
             params.page = $location.search().page || 1;
             params['per_page'] = $location.search()['per_page'] || entriesPerPage;
 
+            //Additional param to override initial load
+            self.overrideAutoLoad = params.overrideAutoLoad || false;
```

Comment:
I am still not sure if I like this being attached to params. That still feels like an abuse of the intention of this parameter to make the coding easier. @waldenraines thoughts?